Documentation/subcommands/stop.md

```diff
@@ -0,0 +1,11 @@
+# rkt stop
+
+Given a list of pod UUIDs, rkt stop will shut them down cleanly.
+
+```
+# rkt stop 387fc8eb cbbf5c01
+"387fc8eb-eabd-4e77-b080-d8c0001eb50c"
+"cbbf5c01-dd52-4ccc-a1e0-cfd8f1e88418"
+# rkt stop --force 93e516b0
+"93e516b0-e84b-40cf-a45b-531b14dfcce2"
```

Comment:
Yeah, but systemd has means to do that synchronously. Doing that with signals is not possible.

Of course I can check if the pod is stopped and retry, or wait until it is stopped. If it's still running after some reasonable amount of time, I can bail out and tell the user to try `--force`.